azkaban-execserver/src/package/conf/azkaban.properties

```diff
@@ -24,3 +24,7 @@ executor.flow.threads=30
 # JMX stats
 jetty.connector.stats=true
 executor.connector.stats=true
+
+# uncomment to enable inmemory stats for azkaban
+#executor.metric.reports=true
+#executor.metric.interval.default=60000
```

Comment:
What is the unit for this prop.? Is it in seconds?  Can we add the unit to the prop. name somehow?